client/analytics/report/taxes/utils.js

```diff
@@ -0,0 +1,8 @@
+/** @format */
+/**
+ * External dependencies
+ */
+
+export function getTaxCode( tax ) {
+	return [ tax.country, tax.state, tax.name ].join( '-' );
```

Comment:
When `tax.country` or `tax.state` are blank this shows several dashes in a row:

![image](https://user-images.githubusercontent.com/3616980/49539482-5f723000-f893-11e8-9922-deb838a3d6c5.png)

How does it sound filtering the strings before joining them?
```suggestion
	return [ tax.country, tax.state, tax.name ].filter( Boolean ).join( '-' );
```

Reply:
Thanks!  Seems like a smart move.  Also trimmed here and modified to match the tax codes in WC, adding 'TAX' as the name if none is defined and appending the priority to the end of the tax code.